mmv1/products/saasservicemgmt/RolloutKind.yaml

```diff
@@ -0,0 +1,153 @@
+          without pausing the rollout.
+  - name: etag
+    type: String
+    description: |-
+      An opaque value that uniquely identifies a version or
+      generation of a resource. It can be used to confirm that the client
+      and server agree on the ordering of a resource being written.
+    output: true
+  - name: labels
+    type: KeyValueLabels
+    description: |-
+      The labels on the resource, which can be used for categorization.
+      similar to Kubernetes resource labels.
+  - name: name
+    type: String
+    description: |-
+      Identifier. The resource name (full URI of the resource) following the standard naming
+      scheme:
+
+      "projects/{project}/locations/{location}/rolloutKinds/{rollout_kind_id}"
+    output: true
+  - name: rolloutOrchestrationStrategy
+    type: String
+    description: |-
+      The strategy used for executing a Rollout. This is a required field.
+
+      There are two supported values strategies which are used to control
+      - "Google.Cloud.Simple.AllAtOnce"
+      - "Google.Cloud.Simple.OneLocationAtATime"
+
+      A rollout with one of these simple strategies will rollout across
+      all locations defined in the associated UnitKind's Saas Locations.
+  - name: uid
+    type: String
+    description: |-
+      The unique identifier of the resource. UID is unique in the time
+      and space for this resource within the scope of the service. It is
+      typically generated by the server on successful creation of a resource
+      and must not be changed. UID is used to uniquely identify resources
+      with resource name reuses. This should be a UUID4.
+    output: true
+  - name: unitFilter
+    type: String
+    description: |-
+      CEL(https://github.com/google/cel-spec) formatted filter string against
+      Unit. The filter will be applied to determine the eligible unit population.
+      This filter can only reduce, but not expand the scope of the rollout.
+  - name: unitKind
+    type: String
+    description: |-
+      UnitKind that this rollout kind corresponds to. Rollouts stemming from this
+      rollout kind will target the units of this unit kind. In other words, this
+      defines the population of target units to be upgraded by rollouts.
+    immutable: true
+    required: true
+  - name: updateTime
+    type: String
+    description: |-
+      The timestamp when the resource was last updated. Any
+      change to the resource made by users must refresh this value.
+      Changes to a resource made by the service should refresh this value.
+    output: true
+  - name: updateUnitKindStrategy
+    type: Enum
+    description: |-
+      The config for updating the unit kind. By default, the unit kind will be
+      updated on the rollout start.
+      Possible values:
+      UPDATE_UNIT_KIND_STRATEGY_ON_START
+      UPDATE_UNIT_KIND_STRATEGY_NEVER
+    enum_values:
+      - "UPDATE_UNIT_KIND_STRATEGY_UNSPECIFIED"
```

Comment:
```suggestion
```

APIs turn this into null (not returned) on response, no need to allow sending it.